core/core_modules/admin.js

```diff
@@ -0,0 +1,176 @@
+﻿var config = require.once('../config.js'),
+
+    getHasPermission = function (config, userId, name, threadId, moduleName) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (12)](https://www.codacy.com/app/theprojectshop/Kassy/file/2573600580/issues/source?bid=3282987&fileBranchId=3292072#l3)